### DISCUSSION TITLE
Please support rendering emoji in readmes

### DISCUSSION THREAD

**Discussion by joshtriplett** [rust-lang/crates.io#9800] [Feature Requests]
Some crates use emoji between `:` in their README. See https://crates.io/crates/wtransport for an example.

crates.io currently displays the `:emojiname:` unrendered. I'd love to see support for rendering the same emoji supported on github.

**Comment by Turbo87** [comment]
it's relatively easy for users to use actual unicode emojis instead, so this is unlikely to get implemented with high priority given the related complexities of supporting a changing set of emojis and rerendering all readmes just for that 😅

**Reply by joshtriplett** [reply]
It's certainly not a high priority. It's useful for compatibility, though.
